src/Pages/Login/Login.tsx

```diff
@@ -0,0 +1,52 @@
+import styles from './Login.module.css';
+import InputElement from '../../Components/InputElement/InputElement';
+import ButtonElement from '../../Components/ButtonElement/ButtonElement';
+import { FormEvent, useState } from 'react';
+
+export default function Login() {
+  const [mail, setMail] = useState<string | null>('');
+  const [password, setPassword] = useState<string | null>('');
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const formdata = new FormData();
+    formdata.append('identifier', `${mail}`);
+    formdata.append('password', `${password}`);
+
+    try {
+      const response = await fetch('http://server.manu-web.de/auth/local', {
```

Comment:
This will fail if this is deployed on a secured domain like `https://heroku...`. It's not possible to access unsecured resources (`http`) from a secured origin (`https`).
-> Make sure, that `manu-web.de` has a valid certificate and is access via `https`.

Reply:
Okay I'll get an SSL certificate soon and change the url here